frontend/app/components/auth/auth.tsx

```diff
@@ -0,0 +1,198 @@
+/** @jsx createElement */
```

Comment:
It will be good if you use hooks for this component.

Reply:
I just moved the old component in a new place. It will be rewritten in the near future.